docker/core-site.xml

```diff
@@ -4,13 +4,13 @@
 <configuration>
 
   <property>
-    <name>google.cloud.auth.service.account.enable</name>
-    <value>true</value>
+    <name>google.cloud.auth.type</name>
+    <value>SERVICE_ACCOUNT_JSON_KEYFILE</value>
   </property>
-  
+
   <property>
     <name>google.cloud.auth.service.account.json.keyfile</name>
     <value>/gsa-key/key.json</value>
   </property>
-  
```

Comment:
These properties look wrong based on [this example](https://github.com/GoogleCloudDataproc/hadoop-connectors/blob/b0fff1c6e45e6c0bd74fc7ce228a9f86da439ce8/gcs/src/test/resources/core-site.xml#L58-L65)

Reply:
Is this still wrong?

Reply:
Dan fixed this, hence spark tests pass

Reply:
I don't know what to say - seems to work in ci?

Reply:
This is what [`Install.md`](https://github.com/GoogleCloudDataproc/hadoop-connectors/blob/af94c95d817f4dc0ce8c4094ce2445a8cc0a1d37/gcs/INSTALL.md?plain=1#L75-L82) recommends

Reply:
*shrug* guess it's fine then